Price policy

lu40 opened the pull request:
Add the disclaimer explaining that on Openki all prices are suggested/orientation prices. Also make the disclaimer dismissable.

It now appears in the following places:
- Introduction (non dismissable but also less flamboyant)
- Course propose
- Course details
- Event create
- Event details

@sbalmer: In order to make sure the user doesn't get bothered again with this disclaimer, I decided to store the `hideUserPolicy` flag in `localStorage`, `Session` and the `user` document. Does this make sense to you?

closes #862

sbalmer commented:
Looks good to me. Can you fix the merge conflicts? I tried but I got confused about which includes are current now.

1u commented:
Good start. here some of my first thoughts that came up:

- Maybe rather say "Openki" should be open to everyone than "Education"? it feels really strong. but maybe thats good :D...

- The message with it's yellow background feels a bit like an error. green?

- The dismiss is not clearly perceived as belonging to the message.. specially with non-white background (event) -> same or similar background?  ...At least the "read on" i would expect inside the box, but in the same time it's nice together with the dismiss as "the two choices"

lu40 commented:
@sbalmer: fixed.
@1u: I implemented the later two changes you proposed. For the sentence I'd rather stay with _Education_, personally I like having this kind of statement inside the information about prices and it feels honest.
